web/src/invite.ts

```diff
@@ -372,6 +372,11 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
             $(e.target).parent().remove();
         });
 
+        $("#invite_streams_container").on("click", ".pill-container", (e) => {
+            const $stream_pill_input_element = $(e.target).find(".input");
+            $stream_pill_input_element.trigger("click");
+        });
+
```

Comment:
@evykassirer is this a good approach to solve what [alya](https://github.com/zulip/zulip/pull/30578#issuecomment-2221258733) is talking about?

Reply:
I think this CSS change would be better and more directly address the issue:

```
.add_streams_container .input {
    flex-grow: 1;
}
```

Reply:
Done.